packages/stryker-webpack-transpiler/src/WebpackTranspiler.ts

```diff
@@ -4,7 +4,7 @@ import WebpackCompiler from './compiler/WebpackCompiler';
 import ConfigLoader from './compiler/ConfigLoader';
 import { setGlobalLogLevel } from 'log4js';
 
-const DEFAULT_STRYKER_WEBPACK_CONFIG = Object.freeze({ configFile: 'webpack.config.js', silent: true });
+const DEFAULT_STRYKER_WEBPACK_CONFIG = Object.freeze({ configFile: 'webpack.config.js', silent: true, context: process.cwd() });
 
 export default class WebpackTranspiler implements Transpiler {
```

Comment:
I think we should still crash is we configure a config file location to a file that does not exist + integration test